models/lyric/train_fasttext.py

```diff
@@ -110,48 +117,58 @@ def create_dataset(replace_newline):
 
     train, test = train_test_split(en_dataset, test_size=0.3, random_state=SEED)
 
+    test, valid = train_test_split(test, test_size=0.5, random_state=SEED)
+
     dataset_path = os.path.join('database', 'fasttext')
     if not os.path.exists(os.path.join(dataset_path)):
```

Comment:
It's the easier way: os.makedirs(path, exist_ok = True)

Reply:
resolved